src/test/java/pingmebot/ParserTest.java

```diff
@@ -0,0 +1,75 @@
+package pingmebot;
+
+import org.junit.jupiter.api.Test;
+import pingmebot.command.*;
+import pingmebot.task.Deadline;
+import pingmebot.task.Events;
+import pingmebot.task.ToDos;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void parseToDoCommandTest() throws PingMeException {
```

Comment:
Could follow the snake case naming convention so it is easier to see what is the input and expected output for each test.